CHANGES.rst

```diff
@@ -19,6 +19,8 @@ Bug Fixes
 
 - Fix string literal generation for SQL query when using numpy >=2.0. [#624]
 
+- Try minimal VOSI tables downloads by default for TAP metadata [#634]
```

Comment:
Rephrasing changelog as this is not default behaviour that the user can change but a change in behaviour.

```suggestion
- Switch to do minimal VOSI tables downloads for TAP metadata. [#634]
```